ext/reflection/php_reflection.stub.php

```diff
@@ -569,6 +569,10 @@ public function hasHook(PropertyHookType $type): bool {}
     public function getHook(PropertyHookType $type): ?ReflectionMethod {}
 
     public function isFinal(): bool {}
+
+    public function isReadable(?string $scope = 'static'): bool {}
```

Comment:
Thinking about this some more, it would likely be beneficial to also accept the object here and check it for `IS_UNDEF`.

Reply:
What `IS_UNDEF` means in userland?

Reply:
`unset()` properties or typed properties that have not been assigned to.

Reply:
Then yes, it would be great to have an object parameter here.